### PR TITLE
Fix the behavior of the TitleWithDescription component tooltip

### PR DESCRIPTION
**Pull request by OliaHrytsak**
Chacked components that use TitleWithDescription component and as a result - only IconExtensionWithTitle.tsx use isDescriptionTooltip(true)
Refactored TitleWithDescription component for conditional rendering of tooltip title without state and handler
Added test for checking tooltip display

image

### CODE REVIEW

**Review by asya982 — APPROVED** [review]

**Review by D1nviz — APPROVED** [review]

File: src/components/title-with-description/TitleWithDescription.tsx [hunk 1]
```diff
@@ -23,24 +23,15 @@ const TitleWithDescription = ({
...
-        </Tooltip>
-      ) : (
-        <Typography onClick={handleTooltip} sx={style.description}>
-          {description}
-        </Typography>
-      )}
+      <Tooltip
+        placement='bottom'
+        title={isDescriptionTooltip ? description : ''}
+      >
```
Comment (mhevyk) [inline src/components/title-with-description/TitleWithDescription.tsx:32]: Is there a need to use `arrow` prop for tooltip here? How do you think?

File: src/components/img-title-description/ImgTitleDescription.tsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ const ImgTitleDescription: FC<ImgTitleDescriptionProps> = ({
 
       <TitleWithDescription
         description={description}
-        style={style.titleWithDescription}
+        style={styles.titleWithDescription}
```
[the remark this answers is not in the document]
Comment (OliaHrytsak) [reply src/components/img-title-description/ImgTitleDescription.tsx]: Fixed! Thanks)

File: src/components/title-with-description/TitleWithDescription.tsx (hunk 1 above)
Comment (OliaHrytsak) [reply src/components/title-with-description/TitleWithDescription.tsx:34]: Good idea! Fixed!

**Review by mhevyk — APPROVED** [review]

**Review by iva-stasia — APPROVED** [review]

File: src/components/title-with-description/TitleWithDescription.tsx [hunk 3]
```diff
@@ -23,24 +24,16 @@ const TitleWithDescription = ({
...
-          </Typography>
-        </Tooltip>
-      ) : (
-        <Typography onClick={handleTooltip} sx={style.description}>
-          {description}
-        </Typography>
-      )}
+      <Tooltip
+        arrow
+        placement='bottom'
```
Comment (Tolik170) [inline src/components/title-with-description/TitleWithDescription.tsx:32]: use enum for 'bottom'

**Review by Tolik170 — APPROVED** [review]

**Review by byavdoshniak — APPROVED** [review]

**Pull request merged**